RecoVertex/PrimaryVertexProducer/python/OfflinePrimaryVertices_cfi.py

```diff
@@ -1,11 +1,23 @@
 import FWCore.ParameterSet.Config as cms
 
-offlinePrimaryVertices = cms.EDProducer("PrimaryVertexProducer",
+DA2DParameters = cms.PSet(
```

Comment:
define elsewhere.
cfi files are not meant for definition of multiple things.
cff may be good